site/public/js/server.js

```diff
@@ -2139,6 +2171,10 @@ function escapeHTML(str) {
     return str.replace(/&/g,'&amp;').replace(/</g,'&lt;').replace(/>/g,'&gt;');
 }
 
+function changePermission(filename, checked) {
+    // send to server to handle file permission change
+    window.location = buildUrl({'component': 'misc', 'page': 'modify_course_materials_file_permission', 'filename': filename, 'checked': checked});
+}
```

Comment:
It might be better to do use an ajax call here instead of changing window.location so that the page doesn't refresh every time a checkbox is clicked.

Reply:
ignore this, next pr as discussed in meeting